frontend/src/utils/format.ts

```diff
@@ -259,3 +259,38 @@ export const calculateActualValue = (value: string, decimals: number) => {
 	// return the actual value as a string
 	return actualValue
 }
+// calculates percentage for BigInt values
+// value is a string representing a BigInt, total is a BigInt
+// returns a percentage as a number (0-100)
+export const calculatePercentageforBigInt = (value: string, total: bigint) => {
+	// value is bigint in string format
+	const valueBigInt = BigInt(value)
```

Comment:
Suggestion would be to maybe just use the type directly as I anyway assume we have the value in bigint format rather than a string format. 

```suggestion
export const calculatePercentageforBigInt = (value: bigint, total: bigint) => {
```